app/models/work_package_types/patterns/token_property_mapper.rb

```diff
@@ -110,7 +120,12 @@ def prefixed_label(context, attribute_label)
       end
 
       def tokenize(custom_field_scope, prefix = nil)
-        custom_field_scope.pluck(:name, :id).map do |name, id|
+        custom_field_scope.pluck(:name, :id, :multi_value).map do |name, id, multiple|
+          formatter = if multiple
+                        ARRAY
+                      else
+                        ->(v) { v.is_a?(Symbol) ? v : v&.to_s }
```

Comment:
❓ Why do we need a separate formatter for custom fields? couldn't it be `STRING_OR_NIL`? or, even better, check the type and map it, because we also have date custom fields, right?

Reply:
> because we also have date custom fields, right?

Do we? That's a good point... Oh dear...

Essentially we needed the special formatter here, because we need to pass the `:attribute_not_available` through unchanged. That's why the special case for symbols.

I was hoping that we'd cover all other cases with `to_s`, but maybe I was wrong with that.